private_notices/views.py

```diff
@@ -23,3 +24,18 @@ def post(self, request):
 		user = request.data.get('userid')
 		return "form"
 '''
+
+class NotificationView(APIView):
+	permission_classes = (IsAuthenticated,IsPrivateNoticeOwner,)
+
+	def get(self, request, user_id):
+		try:
+			user = User.objects.get(id=user_id)
+			print user.username
```

Comment:
@DESHRAJ  Nopes. Debug statements are helpful :smile: 

PS: will remove it no worries :smile: